nservicebus/aws/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: AWS local development
+summary: Guidance on local development when using AWS
```

Comment:
Although this won't be in the nav, so it would be hard to get here without manually altering the URL, I would imagine that any other article added to this section would not be about local development but more AWS stuff in general. Shouldn't this title/summary be more general as well?

Reply:
I don't like the need of this `index` page. I see two options:

1. Delete this page and the aws folder. Move the `nservicebus/aws/local-development.md` under `architecture/aws/local-development.md` and keep the menu links as is pointing to the new location
2. Delete this page and the aws folder. Move the `nservicebus/aws/local-development.md` under `transports/sqs/local-development.md` and keep the menu links as is pointing to the new location

Reply:
I prefer #2. It's a greater affinity for SQS. If you're using Dynamo it should be because you're already using Dynamo and you're really past the point of developing a plan for these things.

Reply:
@DavidBoike — @saratry and I thought the index page could look like a landing/summary page for supported AWS services. What do you think?